Database/Repositories/Implementations/RepositoryBase.cs

```diff
@@ -43,6 +43,12 @@ public virtual async Task<IEnumerable<T>> CreateAsync(IEnumerable<T> entities)
 
     public virtual async Task<T?> GetAsync(int id) => await _context.Set<T>().FindAsync(id);
 
+    public async Task<ICollection<T>> GetAsync(IEnumerable<int> ids) =>
```

Comment:
```suggestion
    public virtual async Task<ICollection<T>> GetAsync(IEnumerable<int> ids) =>
```
Ah ok the previous comment now makes sense.